torch/_export/__init__.py

```diff
@@ -0,0 +1,228 @@
+import contextlib
+import copy
+from typing import Callable, Tuple, Generator, Dict
+
+import torch
+import torch._dynamo as torchdynamo
+from torch._decomp import core_aten_decompositions
+from torch._dispatch.python import enable_python_dispatcher
+from torch.nn.utils import stateless
+from torch.utils import _pytree as pytree
+
+from torch._functorch.aot_autograd import (
+    AOTConfig,
+    create_aot_dispatcher_function,
+    default_partition,
+    run_functionalized_fw_and_collect_metadata,
+)
+
+from torch.fx.experimental.proxy_tensor import (
+    get_proxy_slot,
+    get_torch_dispatch_modes,
+    has_proxy_slot,
+    make_fx,
+    ProxyTorchDispatchMode,
+    set_proxy_slot,
+)
+
+from torch._functorch.eager_transforms import _unwrap_all_tensors_from_functional
+
+from .workflow import ExportedProgram
+
+CORE_ATEN_DECOMPOSITIONS_TABLE = core_aten_decompositions()
+
+
+@contextlib.contextmanager
+def _using_dynamo_config(**kwargs):
```

Comment:
This is a weird util to have in an __init__.py

Reply:
Yeah i will remove this.